modules/oe_bootstrap_theme_paragraphs/src/EventSubscriber/IconOptionsSubscriber.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_bootstrap_theme_paragraphs\EventSubscriber;
+
+use Drupal\oe_paragraphs\Event\IconOptionsEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Provides options for the icon field.
+ */
+class IconOptionsSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents(): array {
+    return [
+      IconOptionsEvent::class => ['getIconOptions', -1],
+    ];
+  }
+
+  /**
+   * Gets the icon options.
+   *
+   * @param \Drupal\custom_events\Event\IconOptionsEvent $event
+   *   Allowed format event object.
```

Comment:
The namespace for the type in the `@param` doc is wrong. It should be `\Drupal\oe_paragraphs\Event\IconOptionsEvent`.